--- ncl-6.5.1/config/LINUX	2018-07-15 18:43:16.000000000 -0700
+++ patch-files/LINUX	2018-10-17 12:26:44.907148273 -0700
@@ -32,8 +32,8 @@
 #define LibSearchUser    -L/usr/X11R6/lib64
 #define IncSearchUser    -I/usr/X11R6/include
 
-#define ArchRecLibSearch    -L/usr/X11R6/lib64
-#define ArchRecIncSearch    -I/usr/X11R6/include
+#define ArchRecLibSearch    -L/usr/X11R6/lib64 -LROLLHDF5/lib -LROLLSZIP/lib
+#define ArchRecIncSearch    -I/usr/X11R6/include -I/usr/include/freetype2 -IROLLNETCDF/include -IROLLHDF4/include -IROLLHDF5/include -IROLLMPI/include
 
 FC = $(F77)
 
